features/energy.py

```diff
@@ -28,7 +28,12 @@ def EnergyJob(crp, energy_options=None, **kwargs):
     )
 
 
-def energy_flow(without_samples=False, samples_options={}, fft_options={}):
+def energy_flow(
+    without_samples=False,
+    normalization_type="divide-by-mean",
+    samples_options={},
+    fft_options={},
```

Comment:
```suggestion
    samples_options={},
    fft_options={},
    normalization_type="divide-by-mean",
```
this comment for your convenience.